aljuarismi/dimensionality.py

```diff
@@ -7,29 +7,118 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import click
 import khiva as kv
 
 import aljuarismi as al
 
 
-def reduce_datapoints(timeseries, parameters):
+def get_int_number(parameters):
     """
-
-    :param timeseries:
-    :param num_points:
-    :return:
+    Obtains or asks for a integer number.
+    :param parameters: the parameters of the function(number,...).
+    :return: An integer number.
     """
-    if timeseries.columns.size > 1:
-        timeseries = timeseries[al.obtain_column(timeseries)]
-    if parameters.get("number") == '':
+    if parameters["number"]:
+        num_points = int(parameters["number"])
+    else:
         print('How many point do you want to reduce it?')
-        num_points = click.prompt('', type=int)
-        click.echo('DEBUG: %d' % num_points)
+        al.voice('How many point do you want to reduce it?')
+        query = al.query_input()
+        while not query.isnumeric():
+            print('Incorrect input.\nIt is not a number.\nPlease introduce one.')
+            al.voice('Incorrect input.\nIt is not a number.\nPlease introduce one.')
+            query = al.query_input()
+        num_points = int(query)
+    return num_points
+
+
+def get_number(parameters):
```

Comment:
Why two functions to get a number? one for integers and one for floats?, if that is the case, please put a proper name like get_float_number